pythonosc/slip.py

```diff
@@ -0,0 +1,82 @@
+#  Copyright (c) 2020. Ruud de Jong
```

Comment:
was this copied verbatim? that github project seems to be MIT so the MIT license should appear somewhere where the copy ended up, please copy it here if so.

Reply:
Added the license info